esmvaltool/cmorizers/obs/cmorize_obs_haloe.ncl

```diff
@@ -0,0 +1,158 @@
+; #############################################################################
+;  ESMValTool CMORizer for HALOE data
+; #############################################################################
+;
+; Tier
+;    Tier 2: other freely-available dataset.
+;
+; Source
+;    Grooss, J.-U. and Russell III, J. M., Atmos. Chem. Phys., 5, 2797-2807,
+;    doi:10.5194/acp-5-2797-2005, 2005.
+;
+; Last access
+;    20200311
+;
+; Download and processing instructions
+;    Download and untar the supplementary material of the above reference.
+;
+; Caveats
+;    The time period includes incomplete years. According to the given
+;    reference (Section 3): "Between October 1991 and August 2002, a total of
+;    78600 HALOE profiles of O3, CH4, H2O, HCl, and HF mixing ratio were then
+;    combined into bins of 5 degree equivalent latitude for 22 pressure levels
+;    between 316 and 0.1 hPa and 12 months".
+;    In order to create T3M fields a fake longitude coordinate is added.
+;
+;  Modification history
+;     20200311-righi_mattia: ported from v1.
+;
+; #############################################################################
+loadscript(getenv("esmvaltool_root") + \
+           "/esmvaltool/cmorizers/obs/interface.ncl")
+
+begin
+
+  ; Script name (for logger)
+  DIAG_SCRIPT = "cmorize_obs_haloe.ncl"
+
+  ; Source name
+  OBSNAME = "HALOE"
+
+  ; Tier
+  TIER = 2
+
+  ; Period
+  YEAR1 = 1991
+  YEAR2 = 2002
+
+  ; Selected variable (standard name)
+  VAR = (/"hus", "tro3"/)
+
+  ; Name in the raw data
+  NAME = (/"H2O", "O3"/)
+
+  ; MIP
+  MIP = (/"Amon", "Amon"/)
+
+  ; Frequency
+  FREQ = (/"mon", "mon"/)
+
+  ; CMOR table
+  CMOR_TABLE = getenv("cmor_tables") + \
+    (/"/cmip5/Tables/CMIP5_Amon"/)
+      "/cmip5/Tables/CMIP5_Amon"/)
+
+  ; Type
+  TYPE = "sat"
+
+  ; Version
+  VERSION = "1"
+
+  ; Global attributes
+  SOURCE = "https://www.atmos-chem-phys.net/5/2797/2005/" + \
+    "acp-5-2797-2005-supplement.tar"
+  REF = " Grooss and Russell, Atmos. Chem. Phys., " + \
+    "doi:10.5194/acp-5-2797-2005, 2005."
+  COMMENT = ""
+
+end
+
+begin
+
+  ; Open input file
+  f = addfile(input_dir_path + "haloe_eqlat_monthly.nc", "r")
+
+  do vv = 0, dimsizes(VAR) - 1
+
+    log_info("Processing " + VAR(vv) + " (" + MIP(vv) + ")")
+
+    time = create_timec(YEAR1, YEAR2)
+    date = cd_calendar(time, 1)
+
+    xx = f->$NAME(vv)$
+
+    ; Special case hus: convert mixing ratio to specific humidity
+    if (VAR(vv).eq."hus") then
+      xx = xx * 18.015 / 28.97  ; [mol/mol] --> [kg/kg]
+      xx = xx / (1. + xx)
+    end if
```

Comment:
@axel-lauer @hb326 
Can you please cross-check the unit conversion?
See [here](https://www.e-education.psu.edu/meteo300/node/519) for example.

Reply:
This looks correct. E.g. for CMIP6, it is explicitely in the CMOR tables that hus is given in kg (H2O) per kg (moist air):
`"comment": "Specific humidity is the mass fraction of water vapor in (moist) air."`
so the conversion of the HALOE data from m3/m3 to kg(H2O)/kg(moist air) as done here is correct.

Reply:
Thanks! Please approve so I can merge.